agent/Gopkg.lock

```diff
@@ -540,6 +540,7 @@
     "github.com/aws/aws-sdk-go/aws/session",
     "github.com/aws/aws-sdk-go/aws/signer/v4",
     "github.com/aws/aws-sdk-go/private/model/api",
+    "github.com/aws/aws-sdk-go/private/protocol",
```

Comment:
Wondering if version pulled by dep ensure before was actually 1.36.0, why this import added?